google/cloud/storage/oauth2/service_account_credentials_test.cc

```diff
@@ -143,17 +143,17 @@ void CheckInfoYieldsExpectedAssertion(ServiceAccountCredentialsInfo const& info,
 TEST_F(ServiceAccountCredentialsTest,
        RefreshingSendsCorrectRequestBodyAndParsesResponse) {
   auto info = ParseServiceAccountCredentials(kJsonKeyfileContents, "test");
-  ASSERT_TRUE(info.ok()) << "status=" << info.status();
+  ASSERT_TRUE(info) << "status=" << info.status();
```

Comment:
I believe we now have a new macro named `ASSERT_STATUS_OK`, that works with both Status and StatusOr objects. It will check that they're OK, and handle logging the appropriate message if not. So you can change this (and remove the streamed status), to just

```cc
ASSERT_STATUS_OK(info);
```

Reply:
I think a lot of this PR has turned into nit-picking about updated `Status`/`StatusOr` semantics,  but I've mostly been rewriting old tests and sticking with the styles used in the rest of the test files.  At this point, I'd prefer moving the status-usage-related stuff into its own separate PR, so we can update entire files at once and not mix logical fixes in the same PR.

Reply:
SGTM. Thanks.